cmake/oeedl_file.cmake

```diff
@@ -9,31 +9,53 @@
 # Usage:
 #
 #       oeedl_file(
-#               <edl_file> <type> <out_files_var>
+#               <edl_file> <type> <out_files_var> [--edl-search-dir dir]
 #
 # Arguments:
 # edl_file - name of the EDL file
-# type - type of files to genreate ("enclave" or "host")
+# type - type of files to genreate ("enclave" or "host" or "enclave-headers" or "host-headers")
 # out_files_var - variable to get the generated files added to
-#
+# --edl-search-dir dir - Additional folder to look for imported edl files.
```

Comment:
"Additional folder **relative to the source directory** to look for imported edl files."

Also, does `oeedger8r --help` need updating as well, or does it pull it from here?